test/e2e/egress_firewall.go

```diff
@@ -290,6 +305,37 @@ spec:
 
 		}
 
+		ginkgo.By("Selecting additional IP addresses for serverNode (networking routing to secondaryIP address on other nodes is harder to achieve)")
+		// add new secondary IP from node subnet to all nodes, if the cluster is v6 add an ipv6 address
```

Comment:
but it looks like you only add it to the server node, not all nodes?

Reply:
yeah fixed this.